src/components/AppNavigationCaption/AppNavigationCaption.vue

```diff
@@ -1,6 +1,15 @@
 <template>
 	<li class="app-navigation-caption">
-		{{ title }}
+		<!-- Title of the caption -->
+		<div class="app-navigation-caption__title">
+			{{ title }}
+		</div>
+
+		<!-- Actions -->
+		<div class="app-navigation-caption__actions">
+			<!-- @slot Slot for the actions menu -->
+			<slot name="actions" />
```

Comment:
The `AppNavigationItem` is automatically expecting some `<ActionXxxx>` components, not an `<Actions>`, for consistency, we should make sure this behaves the same I think :thinking: